src/main/java/com/ustudent/resquod/controller/PositionController.java

```diff
@@ -32,4 +32,26 @@ public String addNewRoom(@RequestBody Position newPosition) {
         }
         return "Position added successfully.";
     }
+
+    @ApiOperation(value = "Change position data", authorizations = {@Authorization(value = "authkey")})
+    @ApiResponses(value = {@ApiResponse(code = 200, message = "Successfully updated!"),
+            @ApiResponse(code = 400, message = "Invalid input!"),
```

Comment:
powinieneś wypisać wszystkie możliwe wiadomości z racji tego że front oraz mobilka będzie mogła od razu to wyświetlać